main-app/app/less/main.less

```diff
@@ -12,4 +12,71 @@ a{
   a:hover {
     color: @link-lowlight-color;
   }
+}
+
+.select {
+  display: inline-block;
+}
+
+//Styles for the table
+table tr td {
+  width: 105px;
+  height: 105px;
+}
+
+.rowBorder {
+  border-top: 1px solid @link-color;
+  border-bottom: 1px solid @link-color;
+}
+
+.cellBorder {
+  border-left: 1px solid @link-color;
+  border-right: 1px solid @link-color;
+}
+
+
+
+//Styles for the image sprites
+//.player1, .player2 {
+//  background: url('../images/noughtsAndCrosses.png') no-repeat;
```

Comment:
Comments again - the section headers in less files are a good idea, the organise things a bit and are removed at compile. 
But the commented source is a problem - in a real project someone will need to maintain & work out why you've left them here.